src/components/code-snippet/_code-snippet.scss

```diff
@@ -299,6 +299,18 @@
 }
 
 @mixin snippet--x {
+  .#{$prefix} {
```

Comment:
Small nit, for new styles we're trying to keep selectors un-nested for discoverability later on in the code lifecycle. Would you mind un-nesting this block to help follow this?

Reply:
Absolutely - I thought that might come up!